rotkehlchen/globaldb/handler.py

```diff
@@ -885,16 +885,16 @@ def get_evm_tokens(
         return tokens
 
     @staticmethod
-    def get_tokens_mappings(addresses: list[ChecksumEvmAddress]) -> dict[ChecksumEvmAddress, str]:  # noqa: E501
-        """Gets mappings: address -> name for tokens whose address is in the provided list"""
-        questionmarks = ','.join('?' * len(addresses))
+    def get_token_name(address: ChecksumEvmAddress, chain_id: ChainID) -> Optional[str]:
```

Comment:
Same bug with token name search. Just simplifying this function that's used only in one place.